### PR TITLE
re-add settings pixels

### PR DESCRIPTION
**Pull request by brindy**
Task/Issue URL: https://app.asana.com/0/392891325557410/1207818776591137/f
Tech Design URL:
CC:

**Description**:
Add pixels for various actions and views in settings.

**Steps to test this PR**:
1. Go through the list of pixels in the linked Asana task and check they fire at the correct time
2. A pixel fired when a view is opened should not fire again if the user pops the stack back to that view
3. None of the pixels should have any addition parameters other than the default (ie app version)

### CODE REVIEW

File: Core/PixelEvent.swift [hunk 1]
```diff
@@ -776,6 +778,26 @@ extension Pixel {
...
+        case settingsSyncOpen
+        case settingsAppearanceOpen
+        case settingsThemeSelectorPressed
+        case settingsAddressBarTopSelected
+        case settingsAddressBarBottomSelected
+        case settingsShowFullURLOn
+        case settingsShowFullURLOff
+        case settingsDataClearingOpen
+        case settingsFireButtonSelectorPressed
+        case settingsDataClearingClearDataOpen
```
Comment (mallexxx) [inline Core/PixelEvent.swift:794]: `settingsDataClearingClearDataOpen ` pixel is not used anywhere

File: Core/PixelEvent.swift [hunk 2]
```diff
@@ -776,6 +778,26 @@ extension Pixel {
...
+        case settingsAutomaticallyClearDataOn
```
Comment (mallexxx) [inline Core/PixelEvent.swift:795]: `settingsAutomaticallyClearDataOn` is fired for every action or timing parameter change in the Automatically Clear Data settings screen

**Review by mallexxx — CHANGES_REQUESTED** [review]
please see 2 issues I commented inline, otherwise looks good

**Comment by brindy** [comment]
Thanks @mallexxx - fixed both of those.

**Review by mallexxx — APPROVED** [review]
LGTM 👍

**Pull request merged**